packages/vector_graphics_compiler/lib/src/paint.dart

```diff
@@ -61,6 +61,9 @@ class Color {
   /// The blue channel value from 0..255.
   int get b => (0x000000ff & value) >> 0;
 
+  /// The opacity channel value from 0..255.
+  int get o => value >> 24;
```

Comment:
nit: This should be called `a` for alpha channel.

Opacity in dart:ui and flutter_svg revers to a double between `0..1` that is basically `a / 255`.

Reply:
Done